packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
-## NEXT
+## 2.8.3
```

Comment:
The changes to this package should be reverted; we don't artificially constrain allowed versions in the app-facing packages.

Reply:
This has not been addressed.